### ISSUE TITLE
Prior edit view

### ISSUE THREAD

**Issue by rk-kontur** [antoniotejada/obsidian-edit-history#8]
This functionality if really great. Thanks again!!!

Is it possible to render teh edit views presented in the dropdown menu as notes, just like any other note? The current viewmode is source code, I believe.

Even better would be integrating the dropdown into the notes themselves (see other request)

**Comment by antoniotejada** [comment]
There are a few issues with that approach:
- The diff needs to show source code because otherwise there are diffs that don't show because Obsidian hides the formatting characters in the formatted view. Imagine you change some text from **text** to ***text***, the diff algorithm is not capable of understanding that it was the formatting for the whole word that changed, for the diff algorithm you just added two asterisks in unrelated places. The change may have even been split across two different edit history entries, which makes it impossible to show in formatted mode. Same goes for sections, etc.
- Having a dropdown in the main editor and displaying the version in formatted mode can make things confusing, you could start editing a previous version without noticing.

One thing I may do is to add a history list that can be placed in the left side pane, where the file-explorer is. Then you could choose the revision from that list and the main pane would show the diffs, but still in diff mode, not in formatted mode.

I imagine there can also be file formats where showing an unformatted diff doesn't make sense (maybe excalidraw notes?, I haven't played with those), in that case I may add some diff blacklist setting so those notes are never diffed and never shown diffed, maybe shown side by side.

**Comment by antoniotejada** [comment]
Actually de-duping, will keep this open for adding a listview of changes in the main pane and the other for adding a side by side formatted view instead of diff view.

**Comment by Azmoinal** [comment]
In my opinion the weak point of the plugin is the diff view that is not so useful when you rewrite some part of the text.

It could be very useful if it could mimic a view similar to the [version history diff plugin](https://github.com/kometenstaub/obsidian-version-history-diff) even if the diff is limited to two consecutive changes.

Currently when you rewrite a portion of a text, the remove/add characters overlaps each others and it is very difficult to understand the old meaning and the new one. 

It will be more readable if modified rows could be shown side by side instead of currently merged representation,

**Comment by feedhopper** [comment]
Yes, I agree. Personally, I would love to see an option to display two or more versions side by side. This here illustrates pretty well what I mean:

![grafik](https://github.com/antoniotejada/obsidian-edit-history/assets/61922985/5dc9daea-dfc4-46fc-9c3d-ad3ccb032fea)

**Comment by rk-kontur** [comment]
WOW. This is REALLY COOL!

Thank you so much! Loving the new timeline view and side by side mode VERY MUCH!
